Automattic/jetpack
# My Jetpack: Remove unused ProductDetailCard

The file for Product Detail was exportng `{ ProductDetail }` and `ProductDetailCard` as default but `ProductDetailCard` is not used. Only used in a story for StoryBook It is only a wrapper for ProductDetail:

```
export default function ProductDetailCard( { slug } ) {
	return (
		<div className={ styles.card }>
			<ProductDetail slug={ slug } />
		</div>
	);
}
```

Not sure if it's ok to remove, but I got confused with the naming, default export and seeing that the default export is not in use by My Jetpack anywhere. If we're planning to use it somewhere else, let me know
This PR removes its usage

#### Changes proposed in this Pull Request:

* Deletes the component ProductDetailCard
* Moves files from `components/product-detail-card` to `product-detail`
* Makes the default export be the `ProductDetail` component.
* Updates the stories accordingly



#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:






* Checkout this branch, 
* Connect the site, build My Jetpack, visit the page
* Confirm you see interstitials by visiting the route `#/add-scan` for example.
* Run storybook and confirm everything looks good
